### DISCUSSION TITLE
Add deployed version to API

### DISCUSSION THREAD

**Discussion by thickconfusion** [alexjustesen/speedtest-tracker#1401] [Ideas]
I like to monitor the deployed version of my containers. api/speedtest/latest does not contain a deployed version of Speedtest. Can it be added to this endpoint's JSON, or a different endpoint?

It may be as simple as editing https://github.com/alexjustesen/speedtest-tracker/blob/4ff86a9eb6d5465c7cc0b84199b7c452c6687b99/app/Http/Controllers/API/Speedtest/GetLatestController.php#L18 and pointing to a system version already being reported.

**Comment by alexjustesen** [comment]
Going to convert this to an issue so I can track it under the API milestone.

**Reply by alexjustesen** [reply]
Added feature to #1419, closing this discussion.
